cmd/lncli/main.go

```diff
@@ -25,12 +41,37 @@ func getClient(ctx *cli.Context) (lnrpc.LightningClient, func()) {
 	return lnrpc.NewLightningClient(conn), cleanUp
 }
 
+type config struct {
+	SSLCert string `long:"sslcert" description:"path to SSL certificate"`
```

Comment:
An un-related comment: this should read `TLSCertPath`. `SSL` is insecure and shouldn't be used by anyone.